src/components/AttachmentPicker/index.js

```diff
@@ -35,14 +34,9 @@ function AttachmentPicker(props) {
                 type="file"
                 ref={fileInput}
                 onChange={(e) => {
-                    let file = e.target.files[0];
+                    const file = e.target.files[0];
 
                     if (file) {
-                        const cleanName = FileUtils.cleanFileName(file.name);
-                        if (file.name !== cleanName) {
-                            file = new File([file], cleanName);
```

Comment:
hello @alitoshmatov I think simply adding the file type here is a much cleaner fix as per Coding guidelines that require you to make the least changes possible.

Reply:
Adding file type here does fix it, but we need to apply this process for drag and drop, and copy paste as well